src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -75,7 +75,22 @@ public void Dispose()
         public IEnumerable<SonarQubeIssue> GetSuppressedIssues(string projectGuid, string filePath)
         {
             // TODO: Block the call while the cache is being built + handle multi-threading
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)